Resync WPT tests under html/syntax and html/dom

rniwa opened the pull request:
#### 5066b9423d8fb583686881150117db5133381f93
<pre>
Resync WPT tests under html/syntax and html/dom
<a href="https://bugs.webkit.org/show_bug.cgi?id=243644">https://bugs.webkit.org/show_bug.cgi?id=243644</a>

Reviewed by Tim Nguyen.

Updated WPT tests under html/dom/ and html/syntax as of 5c79743a93cfc4ded75d99338225531936da0c56

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/directionality/bdi-element-invalid-dir-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/directionality/bdi-element-invalid-dir-ref.html: Renamed
  from LayoutTests/fast/css/bdi-element-invalid-dir-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/html/dom/directionality/bdi-element-invalid-dir.html: Renamed
  from LayoutTests/fast/css/bdi-element-invalid-dir.html.
* LayoutTests/imported/w3c/web-platform-tests/html/dom/directionality/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/innerHTML-setter-default-namespace-expected.txt: Renamed
  from LayoutTests/fast/parser/innerHTML-setter-default-namespace-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/innerHTML-setter-default-namespace.xhtml: Renamed
  from LayoutTests/fast/parser/innerHTML-setter-default-namespace.xhtml.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/Element.getElementsByTagName-foreign-02.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/ambiguous-ampersand-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/ambiguous-ampersand.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html-integration-point.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/math-parse03.html:
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/253203@main">https://commits.webkit.org/253203@main</a>
</pre>

nt1m commented on LayoutTests/imported/w3c/resources/import-expectations.json:
```diff
@@ -32,6 +32,10 @@
+    "html/dom": "import", 
+    "html/syntax": "import", 
+    "html/syntax/parsing-html-fragments": "import", 
+    "html/syntax/parsing-html-fragments/innerHTML-setter-default-namespace.xhtml": "import", 
```
These are redundant with the entries below (`web-platform-tests/html/*`), can you please undo them?

rniwa replied:
Done.

rniwa committed:
Resync WPT tests under html/syntax and html/dom
https://bugs.webkit.org/show_bug.cgi?id=243644

Reviewed by Tim Nguyen.

Updated WPT tests under html/dom/ and html/syntax as of 5c79743a93cfc4ded75d99338225531936da0c56

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/directionality/bdi-element-invalid-dir-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/html/dom/directionality/bdi-element-invalid-dir-ref.html: Renamed
  from LayoutTests/fast/css/bdi-element-invalid-dir-expected.html.
* LayoutTests/imported/w3c/web-platform-tests/html/dom/directionality/bdi-element-invalid-dir.html: Renamed
  from LayoutTests/fast/css/bdi-element-invalid-dir.html.
* LayoutTests/imported/w3c/web-platform-tests/html/dom/directionality/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/innerHTML-setter-default-namespace-expected.txt: Renamed
  from LayoutTests/fast/parser/innerHTML-setter-default-namespace-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/innerHTML-setter-default-namespace.xhtml: Renamed
  from LayoutTests/fast/parser/innerHTML-setter-default-namespace.xhtml.
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing-html-fragments/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/Element.getElementsByTagName-foreign-02.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/ambiguous-ampersand-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/ambiguous-ampersand.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/html-integration-point.html:
* LayoutTests/imported/w3c/web-platform-tests/html/syntax/parsing/math-parse03.html:
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/253203@main
